wiki/Tournaments/CFB/4/en.md

```diff
@@ -27,9 +27,9 @@ The **Catch French Beginner 4** (***CFB4***) was a beginner French double-elimin
 
 | Placing | Prize(s) |
 | :-: | :-- |
-| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 4 months of osu!supporter tag |
-| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter tag |
-| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 4 months of osu!supporter |
```

Comment:
At this point I can't tell if this is the best order and the "x months of osu!supporter, unique profile badge" items are wrong